SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -416,9 +428,10 @@ def customise_Reco(process,pileup):
     process.pixelTracks.FilterPSet.chi2 = cms.double(50.0)
     process.pixelTracks.FilterPSet.tipMax = cms.double(0.05)
     process.pixelTracks.RegionFactoryPSet.RegionPSet.originRadius =  cms.double(0.02)
-    process.templates.DoLorentz=False
-    process.templates.LoadTemplatesFromDB = cms.bool(False)
-    process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)
+    # use defaults d.k. 2/16
+    #process.templates.DoLorentz=False
+    #process.templates.LoadTemplatesFromDB = cms.bool(False)
+    #process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)

```

Comment:
@dkotlins Could you please apply these changes here https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/SimGeneral/MixingModule/python/SiPixelSimParameters_cfi.py#L37 instead? (I would suggest to use e.g. 800pre6) After #12806 that would be the proper place (ok, it doesn't harm to apply the changes also here, but maybe it is not worth of the effort to have the changes in two places).

Said that I'm starting to think if there is really any need to keep the already-migrated-to-era pieces in `phase1TkCustoms.py`. I guess the question is then if any piece of `phase1TkCustoms.py` will be needed for phase2 (tracker/tracking), or will phase2 be built from scratch using eras? @boudoul @ebrondol do you have any idea?